From 0257e41671d17f7875cab2cdb0ae9fc3e7b071a8 Mon Sep 17 00:00:00 2001
From: Sebastian Lackner <sebastian@fds-team.de>
Date: Tue, 27 Jan 2015 14:50:34 +0100
Subject: include: Remove several duplicate definitions from ntdef.h.

FIXME: Should they removed from winternl.h instead? Whats the proper way
to use both include files at the same time?
---
 include/ntdef.h | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/include/ntdef.h b/include/ntdef.h
index 83ecffd..8b35cef 100644
--- a/include/ntdef.h
+++ b/include/ntdef.h
@@ -23,16 +23,6 @@
 extern "C" {
 #endif
 
-typedef enum _EVENT_TYPE {
-    NotificationEvent,
-    SynchronizationEvent
-} EVENT_TYPE;
-
-typedef enum _TIMER_TYPE {
-    NotificationTimer,
-    SynchronizationTimer
-} TIMER_TYPE;
-
 typedef enum _WAIT_TYPE {
     WaitAll,
     WaitAny,
-- 
2.2.1

